templates/explore/repo_list.tmpl

```diff
@@ -7,6 +7,9 @@
 					{{if $avatar}}
 						{{$avatar}}
 					{{end}}
+					{{if .IsPinned}}
```

Comment:
As mentioned, that is one possible approach, however, I do not know if we want to use this approach in the end.